UI/importers/studio.cpp

```diff
@@ -16,6 +16,7 @@
 ******************************************************************************/
 
 #include "importers.hpp"
+#include <obs-nix-platform.h>
```

Comment:
This header must not be included on Windows and macOS.

Reply:
Added a platform check